src/js/components/help/connecting-devices/raspberry-pi-3.js

```diff
@@ -85,11 +87,27 @@ var RaspberryPi =  createReactClass({
         <p>Wait for 3-4 minutes until the device has fully booted and the Mender client has been able to connect to the Mender server.</p>
         <p>You should see your device pending in the <a onClick={this.props.changePage.bind(null, "devices")}>Devices tab</a>. Simply Authorize the pending device.</p>
 
-        <p>If you cannot see any new device after 10 minutes, please verify that the network connection is working, and feel free to reach out for help with diagnostics via <a onClick={this.props.changePage.bind(null, "help/more-help-resources")}>our further Help resources</a>.</p>
+        <p> If you cannot see any new device after 10 minutes, please verify that the network connection is working, and feel free to reach out for help with diagnostics via <a onClick={this.props.changePage.bind(null, "help/more-help-resources")}>our further Help resources</a>.</p>
+
+        <p> NOTE: If you reimage the device with the disk image or switch storage (e.g. SD cards) between the devices after they have booted once, <b>authenticating the device to the Mender server will fail</b>. This is because the Mender server tracks the identity (MAC address by default) <-> device public key (randomly generated upon first run of the Mender client) binding. If this happens, you need to *decommission* the device from the Mender server and try again. <p>
+
+
+        <h3>Deploy updates</h3>
+
+        {this.props.isHosted ? 
+          <p>Download these two Raspberry Pi 3 Artifacts customized for your Hosted Mender account: TODO LINKS: Artifact1, [Artifact2].</p>
```

Comment:
Needs adjustment for Rpi3 Artifact 1 & Rpi3 Artifact 2 URLs.

Reply:
@estenberg: The `<->` after "(MAC address by default)" causes an issue when running gulp. Is it a typo?